examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -463,10 +466,29 @@ def main():
                 **kwargs,
             )
 
-            pipeline.to(torch.device("hpu"))
             pipeline.unet.set_default_attn_processor(pipeline.unet)
+            pipeline.to(torch.device("hpu"))
+
+            quant_config_path = os.getenv("QUANT_CONFIG")
+            if quant_config_path:
+                import habana_frameworks.torch.core as htcore
+                from neural_compressor.torch.quantization import FP8Config, convert, prepare
+
+                htcore.hpu_set_env()
+
+                config = FP8Config.from_json_file(quant_config_path)
```

Comment:
I suggest we move the json and binary to a repo on the hub and only download it for the example using [`huggingface_hub.hf_hub_download`](https://huggingface.co/docs/huggingface_hub/v0.26.3/en/package_reference/file_download#huggingface_hub.hf_hub_download)

Reply:
Hi @IlyasMoutawwakil thanks for the suggestions, we will add it as part of this JIRA (SW-211352) later, for now lets merge it for the release.

Reply:
Nah, I'll push a new commit today, we should not keep bianries in the repo like this

Reply:
yeh and the change is as simple as:
```suggestion
                config = FP8Config.from_json_file(json.load(open(huggingface_hub.hf_hub_download(
                            repo_id="Habana/stable-diffusion",
                            filename="fp8_hooks_maxabs.json",
                        ), mode="r",
                    )
                )
```